src/gui/sdlmain.cpp

```diff
@@ -1095,6 +1136,23 @@ static void setup_presentation_mode(FRAME_MODE &previous_mode)
 		schedule_synced();
 }
 
+static void NewMouseScreenParams()
+{
+	int abs_x, abs_y;
+	SDL_GetMouseState(&abs_x, &abs_y);
+	if (abs_x < 0 || abs_x > UINT16_MAX || abs_y < 0 || abs_y > UINT16_MAX) {
+		LOG_ERR("SDL: Unexpected mouse position values: %dx%d", abs_x, abs_y);
```

Comment:
Don't you need to clamp the values in this case or return early?

Reply:
This is actually checking for strange values from SDL. I have changed it to assertions, as advised by @kcgen in other remark.